library/core/src/os/darwin/objc.rs

```diff
@@ -67,7 +67,9 @@ pub type SEL = *mut objc_selector;
 ///
 /// # Example
 ///
-/// ```no_run
+// Ferrocene addition: this fails to link. Since it's marked no_run, it doesn't generate coverage
+// anyway. Change it from no_run -> ignore.
+/// ```ignore
```

Comment:
```
tidy [style (library)]: /home/runner/work/ferrocene/ferrocene/library/core/src/os/darwin/objc.rs:72: unexplained "```ignore" doctest; try one:

* make the test actually pass, by adding necessary imports and declarations, or
* use "```text", if the code is not Rust code, or
* use "```compile_fail,Ennnn", if the code is expected to fail at compile time, or
* use "```should_panic", if the code is expected to fail at run time, or
* use "```no_run", if the code should type-check but not necessary linkable/runnable, or
* explain it like "```ignore (cannot-test-this-because-xxxx)", if the annotation cannot be avoided.
```